core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceServiceCloneSpec.scala

```diff
@@ -733,6 +733,38 @@ class MultiCloudWorkspaceServiceCloneSpec
     result.errorReport.statusCode.value shouldBe StatusCodes.Forbidden
   }
 
+  it should "throw an exception for an Azure workspace if the workspaceRequest contains a nonempty authorizationDomain" in {
```

Comment:
Do we have test coverage of cloning and creating GCP workspaces with an authorizationDomain?

Reply:
Yeah, there are a few tests

- [Checking that the auth domain protected workspace bucket has secure logging after creating](https://github.com/broadinstitute/rawls/blob/f507389df6eefc8711f3394ce71ba57d086b8963/core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala#L2251)
- [Checking that the auth domain protected workspace bucket has secure logging after cloning](https://github.com/broadinstitute/rawls/blob/f507389df6eefc8711f3394ce71ba57d086b8963/core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala#L2654)
- [Checks that MultiCloudWorkspaceService passes GCP clone requests to WorkspaceService](https://github.com/broadinstitute/rawls/blob/f507389df6eefc8711f3394ce71ba57d086b8963/core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceServiceCloneSpec.scala#L1092)
- [API unit test that includes an auth domain for a GCP workspace](https://github.com/broadinstitute/rawls/blob/f507389df6eefc8711f3394ce71ba57d086b8963/core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala#L406)